docs/programming-oak.md

```diff
@@ -685,13 +685,94 @@ TODO: describe use of storage
 
 ## Using External gRPC Services
 
-TODO: describe use of gRPC client pseudo-Node to connect to external gRPC
-services.
+To allow Oak applications connect to external gRPC services, the Oak SDK
+provides a convenient API for creating gRPC Client pseudo nodes:
+
+<!-- prettier-ignore-start -->
+[embedmd]:# (../examples/aggregator/module/rust/src/lib.rs Rust /.*let grpc_client_invocation_sender.*/ /gRPC client.*/)
+```Rust
+        let grpc_client_invocation_sender = oak::grpc::client::init(&config.backend_server_address)
+            .context("Couldn't create gRPC client")?;
+```
+<!-- prettier-ignore-end -->
+
+The client is initialized with the `address` of the external gRPC service (e.g.,
```

Comment:
I think it should be called URI. But I guess it's already called `address` in the code.

Reply:
Yes. It is called address in the code.